OracleCoherence/README.md

```diff
@@ -1,115 +1,199 @@
+Java 8 (see [Oracle Java images](../OracleJava/)) or GraalVM CE (see [GraalVM](../GraalVM/CE/))
+base image.
+
+>Note: The GraalVM CE base image can only be used for Oracle Coherence 14.1.1.0.0.
 
 ## How to Build
 
-For Coherence 12.2.1.4.0, a Maven project is provided to facilitate building the Docker image.  Maven is used to pull dependent libraries
-which are then bundled into the Docker image to enable running Coherence Management over REST and Coherence Metrics endpoints within the container.
+For Coherence 14.1.1.0.0 and 12.2.1.4.0, a Maven project is provided to facilitate building
+the Docker image.  Maven is used to pull dependent libraries which are then bundled into the
+Docker image to enable running Coherence Management over REST and Coherence Metrics endpoints 
+within the container.
+
+### To build a Coherence 14.1.1.0.0 or 12.2.1.4.0 Docker image
 
-### To build a Coherence 12.2.1.4.0 Docker image
+#### Prerequisites for building with Maven
 
-Prerequisites for building with Maven:
 * Java 8 JDK
 * Maven 3.6.1
 
-Build steps:
+#### Build steps
+
+The following steps build a Coherence 14.1.1.0.0 Docker container as an example
 
 1. Checkout the GitHub Oracle Docker Images repository
-   
-   	`$ git clone git@github.com:oracle/docker-images.git`
+
+    ```shell
+    git clone git@github.com:oracle/docker-images.git
+    ````
    	
-2. Go to the directory containing the Dockerfile for 12.2.1.4
+1. Go to the directory containing the Dockerfile
 
-    `$ cd OracleCoherence/dockerfiles/12.2.1.4.0/src/main/docker`
+    ```shell
+    cd OracleCoherence/dockerfiles/14.1.1.0.0/src/main/docker
+    ```
 
-3. [Download](https://www.oracle.com/middleware/technologies/coherence-downloads.html) and drop the Coherence distribution file of your choice into this folder. The build script supports either building an image from either the Standalone Installer, **fmw_12.2.1.4.0_coherence_Disk1_1of1.zip** or the Quick Installer **fmw_12.2.1.4.0_coherence_quick_Disk1_1of1.zip**
+1. [Download the Coherence distribution file](https://www.oracle.com/middleware/technologies/coherence-downloads.html)
+   of your choice and save it to the current directory. The build script supports
+   building an image from either the Quick Installer, `fmw_14.1.1.0.0_coherence_quick_Disk1_1of1.zip`,
+   or the Standalone Installer, `fmw_14.1.1.0.0_coherence_Disk1_1of1.zip`.
 
-    Note that the `Dockerfile` is currently configured to use the Quick Installer.
+    > **_NOTE:_** The `Dockerfile` is currently configured to use the Quick Installer.
 
-4. Go to the maven project directory
+1. Go to the maven project directory
 
-    `$ cd OracleCoherence/dockerfiles/12.2.1.4.0`
+    ```shell
+    cd OracleCoherence/dockerfiles/14.1.1.0.0
+    ```
     
-5. Build the Docker image with Maven
+1. Build the Docker image with Maven
 
-    `$ mvn install`
+    To build a Docker image using Oracle 8 JDK:
```

Comment:
I would repeat the fact that the `oracle/serverjre:8` image should be available prior to running this command.

Reply:
I added an additional step in the build instructions to ensure the image is available (and the Graal image as well if building a Graal based Coherence image).